src/Product.php

```diff
@@ -191,4 +191,13 @@ public function setProducts(array $products = [])
 
         $this->elements['product-set-products'] = $xml;
     }
+
+    public function setImages(string $path, string $viewType = 'large')
```

Comment:
I'm undecided whether this method would be better named `setVisualMerchandising`. Following on from other methods, I've matched the element name it's adding, but it could cause confusion with primary/secondary images, which are set elsewhere.